### DISCUSSION TITLE
Adding more visible hints.

### DISCUSSION THREAD

**Discussion by Rancho-7** [apache/hertzbeat#2879] [Ideas]
When we try to add a new monitor for OS,should we set `Password` or `PrivateKey` as required,like `Username`? 

![image](https://github.com/user-attachments/assets/0972f8e3-38dd-4e0e-8376-96290f86a4b8)

I think this will give the user more friendly hints,because if these two parameters are both not filled in,it will prompt a authentication error.

![image](https://github.com/user-attachments/assets/a9f84c4b-425e-421d-984b-334d8f0a5d59)

I am not sure if it is really needed,so I would like to discuss about it here with everyone.

**Comment by tomsun28** [comment]
Hi, due the either password or PrivateKey is required, it is difficult to determine the display here.
